### ISSUE TITLE
Task - use documentation to setup detox test run

### ISSUE THREAD

**Issue by DJUltraTom** [department-of-veterans-affairs/va-mobile-app#8781]
Ticket to document time I use to set up a detox test run using Rachaels documentation

**Comment by bischoffa** [comment]
@DJUltraTom this shows in New Issues but has a sprint from mid June. Is this ticket in the correct status?

**Comment by timwright12** [comment]
@DJUltraTom @bischoffa What's the status of this ticket?

**Comment by DJUltraTom** [comment]
updated the ticket to contain links to documentation in the description. 
Started following the instructions, hit a roadblock, set up a 1:1 w/ Rachael for tomorrow to get me unblocked.

**Comment by DJUltraTom** [comment]
synced with Rachael and was not able to get unblocked during our allocated time. 
Will follow up in an attempt to get unblocked tomorrow, updated ticket estimate

**Comment by DJUltraTom** [comment]
For clarity of this ticket, I am able to run the detox tests on github without a problem, local environment is presenting some challenges.

**Comment by DJUltraTom** [comment]
moving to blocked until I can get my local environment functioning correctly

**Comment by TKDickson** [comment]
@DJUltraTom @timwright12 I think we can close this ticket as no longer needed, but lmk if either of you disagree.
If the point of this ticket was to test that Rachael's documentation made sense enough for experience teams to do this work - our FE folks have all set up detox using those instructions, on their local machines.
If the point of this ticket was for Tom to get a local setup working to be able to do detox, I'd say chuck that idea and use codespaces instead (and open a new ticket for "I can do stuff with codespaces").
Or was there a different purpose that I'm missing?

**Comment by timwright12** [comment]
Codespaces is an interesting idea. It's set up for the doc site right now, but I don't think there's anything preventing us from running tests... maybe the emulators. Let's keep this open for now
